lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -855,14 +855,14 @@ sub create_artefact {
     }
 
     $asset->move_to(join('/', $storepath, $asset->filename));
-    OpenQA::Utils::log_debug("moved to $storepath " . $asset->filename);
+    log_debug("moved to $storepath " . $asset->filename);
 
     # mark the worker as alive
     if ($self->worker) {
         $self->worker->seen;
     }
     else {
-        OpenQA::Utils::log_warn(printf("%d got an artefact but has no worker. huh?", $self->id));
+        log_warning(sprintf("%d got an artefact but has no worker. huh?", $self->id));
```

Comment:
Because you're missing space before 'got'. :)
Seriously though, I'm for concatenation in this instance. And the next `log_debug` looks funny too.